clar2wasm/src/words/blockinfo.rs

```diff
@@ -227,6 +248,16 @@ mod tests {
 
     //- Block Info
 
+    #[test]
+    fn get_block_info_less_than_two_args() {
+        crosscheck_expect_failure("(get-block-info? id-header-hash)");
+    }
```

Comment:
`crosscheck_expect_failure` is somewhat deprecated. We should avoid using it since we don’t know which error is actually being caught by that function. Please check comments [here](https://github.com/stacks-network/clarity-wasm/blob/main/clar2wasm/src/tools.rs#L688-L693).

Here’s a good example of that. The error being thrown is probably not the expected `ArgumentError`.

Consider using other variations of `crosscheck` functions throught this PR.

Reply:
I updated the tests so they do not include `crosscheck_expect_failure`.

They are now using the `evaluate` function.